SPECS/glusterfs/glusterfs.spec

```diff
@@ -811,196 +1129,245 @@ exit 0
 %if ( 0%{!?_without_server:1} )
 %preun server
 if [ $1 -eq 0 ]; then
-    if [ -f %{glusterfsd_svcfile} ]; then
-        %{service_stop} glusterfsd
+    if [ -f %glusterfsd_svcfile ]; then
+        %service_stop glusterfsd
     fi
-    %{service_stop} glusterd
-    if [ -f %{glusterfsd_svcfile} ]; then
+    %service_stop glusterd
+    if [ -f %glusterfsd_svcfile ]; then
         %systemd_preun glusterfsd
     fi
     %systemd_preun glusterd
 fi
 if [ $1 -ge 1 ]; then
-    if [ -f %{glusterfsd_svcfile} ]; then
+    if [ -f %glusterfsd_svcfile ]; then
         %systemd_postun_with_restart glusterfsd
     fi
     %systemd_postun_with_restart glusterd
 fi
 exit 0
-%endif
 
 %preun thin-arbiter
 if [ $1 -eq 0 ]; then
-    if [ -f %{glusterta_svcfile} ]; then
-        %{service_stop} gluster-ta-volume
+    if [ -f %glusterta_svcfile ]; then
+        %service_stop gluster-ta-volume
         %systemd_preun gluster-ta-volume
     fi
 fi
+%endif
 
 ##-----------------------------------------------------------------------------
 ## All %%postun should be placed here and keep them sorted
 ##
 %postun
-%{?ldconfig}
 %if ( 0%{!?_without_syslog:1} )
-
+%if ( 0%{?fedora} ) || ( 0%{?rhel} && 0%{?rhel} >= 6 )
```

Comment:
Please remove all references to fedora, rhel, etc. thanks

Reply:
i think it should stay.

the more we diverge from the upstream spec.in (by removing these things), the harder it is to update it each time.

Reply:
@neha170 , is this an acceptable answer?

Reply:
out of curiosity, why does it matter that rhel/fedora references exist?

Reply:
I think, generally for aesthetics and reduction of unnecessary code in the spec but maybe there are other reasons that I am not aware of.

Reply:
seems to introduce complication for future upgrades